Quick Stock Balance Documentation (#595)
frappe/erpnext_com: erpnext_com/www/docs/user/manual/en/stock/quick-stock-balance.md

<!-- add-breadcrumbs -->
# Quick Stock Balance

> Quick Stock Balance is a tool that can be used to quickly retrieve the current stock balance (quantity) and value of an item, in the specified warehouse, on the specified date.

Reviewer: ```suggestion
**Quick Stock Balance is a tool that can be used to quickly retrieve the current stock balance (quantity) and value of an Item, in the specified wWarehouse, on the specified date.**
```

It comes in handy during audits, while verifying stock details before transactions, etc.

To access this tool, go to:

> > Home > Stock > Tools > Quick Stock Balance

Reviewer: THIS BREADCRUMB DOES NOT EXIST IN THE PRODUCT

Author: this was discussed :)

> ## 1. Using the tool

Reviewer: ```suggestion
## 1. Using Quick Stock Balance
```

> Item and Warehouse are the basic fields required to fully utilise this tool. Item Barcode can also be used to fetch stock details.

Reviewer: ```suggestion
Item and Warehouse are the basic fields required to fully utilize Quick Stock Balance. Item Barcode can also be used to fetch stock details.
```

![Quick Stock Balance](/docs/assets/img/stock/quick-stock-balance.png)

> You can opt to select a different warehouse  or date and the stock value, along with the quantity, will be retrieved dynamically according to the specified values.

Reviewer: ```suggestion
You can opt to select a different warehouse or date. The stock value and quantity, will be fetched real-time according to the specified values.
```

> You can also view the Stock Balance report by clicking on 'Stock Balance Report' . It will redirect you to the report with the item and warehouse filters populated.

Reviewer: ```suggestion
You can also view the Stock Balance report by clicking on 'Stock Balance Report'. It will redirect you to the report with the Item and Warehouse filters populated.
```

> You can also view the Stock Balance report by clicking on 'Stock Balance Report' . It will redirect you to the report with the item and warehouse filters populated.

Reviewer: add a screenshot for this
